modules/ROOT/pages/advanced/editor-context-menu-identifiers.adoc

```diff
@@ -6,4 +6,4 @@
 
 include::partial$misc/available-context-menu-sections.adoc[]
 
-For information on customizing the context menu and adding custom context menu sections, see: link:{baseurl}/ui-components/contextmenu/#availablecontextmenusections[Context menu].
+For information on customizing the context menu and adding custom context menu sections, see: xref:ui-components/contextmenu.adoc#availablecontextmenusections[Context menu].
```

Comment:
This xref doesn't resolve as the `availablecontextmenusections` id doesn't exist. It looks like the id is missing in `modules/ROOT/partials/misc/available-context-menu-sections.adoc`.

- [ ] Fix broken xref

Reply:
solved